src/gui/main.cpp

```diff
@@ -4,7 +4,7 @@
 #include "../agent.h"
 
 MyFrame::MyFrame(wxWindow* parent, wxWindowID id, const wxString& title, const wxPoint& pos, const wxSize& size, long style):
-    wxFrame(parent, id, title, pos, size, wxDEFAULT_FRAME_STYLE)
+    wxFrame(parent, id, title, pos, size, wxNETVFYDEFAULT)
```

Comment:
nit: I guess you could put them directly there:  "[...], size, flag | flag | flag)"